runtime/include/vendor_version.h

```diff
@@ -19,10 +19,10 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
  *******************************************************************************/
-
```

Comment:
This blank line was helpful in separating the copyright notice from what follows.